bubuku/controller.py

```diff
@@ -192,5 +203,6 @@ def _add_change_to_queue(self, change):
     def stop(self, change: Change):
         _LOG.info('Stopping controller with additional change: {}'.format(change.get_name() if change else None))
         # clear all pending changes
```

Comment:
Was this a `TODO` comment?  I don't see that we clear anything here.

Reply:
I think it should be removed